salt/modules/ansiblegate.py

```diff
@@ -162,9 +167,13 @@ def call(self, module, *args, **kwargs):
             ["echo", "{0}".format(js_args)],
             stdout=subprocess.PIPE, timeout=self.timeout)
         proc_out.run()
+        if six.PY2:
+            proc_out_stdout = proc_out.stdout
+        else:
+            proc_out_stdout = proc_out.stdout.decode()
```

Comment:
Could you please consider to use `proc_out_stdout = salt.utils.stringutils.to_str(proc_out.stdout)` here?

Reply:
Done :+1: